tests/integration/handle.py

```diff
@@ -20,7 +21,11 @@ def handle(event, context):
             record_ids.append(record["Sns"]["MessageId"])
 
     # Generate custom metrics
+    timestamp = time.time() - 60
     lambda_metric("hello.dog", 1, tags=["team:serverless", "role:hello"])
+    lambda_metric(
+        "hello.cat", 1, tags=["team:serverless", "role:hello"], timestamp=timestamp
+    )
```

Comment:
Creating a custom metric with a timestamp forces our instrumentation to use the datadog api package instead of sending the metric to the extension.